### PR TITLE
Propagate CMSSW_12_5_0_pre4 to the web page

### PR DESCRIPTION
**Pull request by xoqhdgh1002**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -6051,10 +6051,6 @@ <h3>Summary of CMSSW_12_2_X(NanoAOD)(11834) <font size="2em"><a target='_blank'
 				<ul>
 				<li>
 				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/Time_Mem_Summary/CMSSW_12_2_0/slc7_amd64_gcc900/136.889/step3.txt" title="getTimeMemSummary">[getTimeMemSummary]</a>
-
-			<li>Circle (Pie chart) :
-			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/circles/piechart.php?local=false&dataset=CMSSW_12_2_0%2Fslc7_amd64_gcc900%2F136.889%2Fstep3_circles&resource=time_real&colours=default&groups=package&threshold=0" title="Circle">[Circle]</a>
-			</li>
```
Comment (jpata) [inline web/index.html:6057]: why are these circles being removed?

**Comment by jpata** [comment]
Thanks for the update. Can you remove the .pyc files from the PR? Also, it looks like there are conflicts which need to be resolved.

File: web/index.html (hunk 1 above)
Comment (xoqhdgh1002) [reply web/index.html:6057]: I think 136.889 doesn't have circles and I removed that.
So It is a correct HTML file.

**Comment by xoqhdgh1002** [comment]
I removed that! check, please!

**Comment by jpata** [comment]
Thanks, that's better.

**Comment by jpata** [comment]
looks like there's still conflicts with the master branch, and this cannot be merged. Can you check the conflicts (merge master into your branch and resolve all issues)?

**Comment by jpata** [comment]
@xoqhdgh1002 could you reopen this after resolving the conflicts? thanks!

**Pull request closed without merging**